project.clj

```diff
@@ -1,7 +1,7 @@
 (def ks-version "1.2.0")
 (def tk-version "1.2.0")
 
-(defproject puppetlabs/trapperkeeper-metrics "0.1.3-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper-metrics "0.2.0-SNAPSHOT"
```

Comment:
Is this the right thing to do to release a Y version? I'm not sure I've ever done a y release from jenkins, so I don't actually have any idea how the release job works for it.